tests/behaviour/steps/inspectcli.py

```diff
@@ -1,12 +1,37 @@
+import pandas as pd
 from difflib import ndiff
 from pathlib import Path
 
 from behave import *
+from pandas.util.testing import assert_frame_equal
+from more_itertools import first
```

Comment:
We already have a function called `first` defined [here](https://github.com/GSS-Cogs/csvcubed/blob/main/src/csvcubed/utils/iterables.py). Can you use that instead so we don't add any new dependencies, even if they are only in testing.